### PR TITLE
New Ice Shards 2

### LINKED ISSUE #1270: Damaging ice chunks and the particles look the same
which might be a bit confusing

### PR DESCRIPTION
**Pull request by FuJa0815**
**Brief Description of What This PR Does**

This PR replaces #1392
It adds a new shader to the ice shards

**Related Issues**

closes #1270
closes #1392 

**Progress Checklist**

Note: before starting this checklist the issue should be marked as non-draft.

- [ ] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author.

### CODE REVIEW

**Review by Kasterisk — COMMENTED** [review]
Screenshots on how this looks like would help a lot.

File: assets/models/Iceshard1.tscn [hunk 1]
```diff
@@ -1,8 +1,26 @@
-[gd_scene load_steps=2 format=2]
+[gd_scene load_steps=4 format=2]
 
 [ext_resource path="res://assets/models/Iceshard1.mesh" type="ArrayMesh" id=1]
 
+[sub_resource type="Shader" id=1]
```
Comment (Kasterisk) [inline assets/models/Iceshard1.tscn:5]: Shader needs to be in its own file which could then be hooked up to the iceshard's ShaderMaterial (this needs to be created anew as the iceshard currently still use a SpatialMaterial)

**Review by Kasterisk — CHANGES_REQUESTED** [review]

**Comment by FuJa0815** [comment]
grafik
grafik

**Comment by hhyyrylainen** [comment]
Doesn't that still look a bit too much like the environmental particles to very easily distinguish between them?

**Comment by hhyyrylainen** [comment]
I suppose if we don't have a modeller to work on a new model, we'll just have to work with what we got and at least this is visually more distinct than the current state in the game...

File: assets/models/Iceshard1.tscn (hunk 1 above)
Comment (hhyyrylainen) [reply assets/models/Iceshard1.tscn:5]: Agreed. I'll do my own review on this PR once this is resolved.

**Comment by hhyyrylainen** [comment]
I suppose this is waiting for the new model by @Nunez2196 to be complete and added here.

**Comment by hhyyrylainen** [comment]
I'm closing this PR as it's been over a year since progress was made on this. Please feel free to reopen this if there is a chance that this will get finished.

**Pull request closed without merging**